java/ql/lib/semmle/code/xml/AndroidManifest.qll

```diff
@@ -18,6 +18,16 @@ class AndroidManifestXmlFile extends XMLFile {
    * Gets the top-level `<manifest>` element in this Android manifest file.
    */
   AndroidManifestXmlElement getManifestElement() { result = this.getAChild() }
+
+  /**
+   * Holds if this Android manifest file is located in a build directory.
+   */
+  predicate isInBuildDirectory() {
+    exists(AndroidManifestXmlFile file |
+      file = this.getFile() and
+      file.getRelativePath().matches("%build%")
+    )
+  }
```

Comment:
This can be simplified:

```suggestion
    this.getFile().getRelativePath().matches("%build%")
```

Reply:
Updated in commit [fe127e5](https://github.com/github/codeql/pull/9939/commits/fe127e5adccc58a83b6c6268212d2a1bba7f4377).